<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>420</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>AddressBook</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout"/>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>411</width>
     <height>421</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>255</green>
         <blue>255</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Base">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="Window">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>232</red>
         <green>232</green>
         <blue>232</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="inputTab">
    <attribute name="title">
     <string>Initial</string>
    </attribute>
    <widget class="QLabel" name="h0Label">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>h(t=0) [m]</string>
     </property>
     <property name="buddy">
      <cstring>h0Line</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="thetaLine">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>72</y>
       <width>114</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="h0Line">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>24</y>
       <width>114</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="thetaLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>53</y>
       <width>79</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Theta(t=0) [K]</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
    </widget>
    <widget class="QLabel" name="dthetaLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>101</y>
       <width>63</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Δϴ(t=0) [K]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dthetaLine">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>120</y>
       <width>114</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="labthetaLabel">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>149</y>
       <width>58</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>γθ [K m-1]</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="labthetaLine">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>168</y>
       <width>114</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="readButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>197</y>
       <width>74</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Read in</string>
     </property>
    </widget>
    <widget class="QPushButton" name="calcButton">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>225</y>
       <width>84</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Calculate</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="outputTab">
    <attribute name="title">
     <string>Output</string>
    </attribute>
    <widget class="QGraphicsView" name="graphicsView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>381</width>
       <height>371</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>thetaLine</tabstop>
  <tabstop>readButton</tabstop>
  <tabstop>calcButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
